src/lib/stores/localStorage.svelte.ts

```
@@ -0,0 +1,97 @@
+			if (localStorage.getItem(key) === null) {
+				localStorage.setItem(key, JSON.stringify(initial));
+			}
+		}
+	}
+
+	get current() {
+		this.#version;
+
+		const root =
+			typeof localStorage !== 'undefined'
+				? JSON.parse(localStorage.getItem(this.#key) as any)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)